src/Api/AdminConsole/Public/Controllers/PoliciesController.cs

```diff
@@ -85,10 +86,10 @@ public async Task<IActionResult> List()
     [ProducesResponseType(typeof(PolicyResponseModel), (int)HttpStatusCode.OK)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [ProducesResponseType((int)HttpStatusCode.NotFound)]
-    public async Task<IActionResult> Put(PolicyType type, [FromBody] PolicyUpdateRequestModel model)
+    public async Task<IActionResult> Put(PolicyType type, [FromBody] ProtoTypeEnvelopeModel<PolicyUpdateRequestModel> model)
```

Comment:
@eliykat Start with this endpoint and see how we can pass it through the system.